packages/graphile-utils/__tests__/makePgSmartTagsPlugin.test.js

```diff
@@ -50,51 +91,59 @@ it("allows adding a custom single field to PG schema", async () => {
   try {
     const { data, errors } = await graphql(
       schema,
-      `
-        query {
-          allUsers {
-            nodes {
-              email
-              testSmartTagByEmail {
-                email
-                val
-              }
-            }
-          }
-        }
-      `,
+      testQuery,
       null,
       { pgClient },
       {}
     );
     expect(errors).toBeFalsy();
     expect(data).toBeTruthy();
-    expect(data).toMatchInlineSnapshot(`
-      Object {
-        "allUsers": Object {
-          "nodes": Array [
-            Object {
-              "email": "alice@example.com",
-              "testSmartTagByEmail": null,
-            },
-            Object {
-              "email": "bob@example.com",
-              "testSmartTagByEmail": Object {
-                "email": "bob@example.com",
-                "val": 42,
+    expect(data).toMatchInlineSnapshot(inlineSnapshot);
```

Comment:
`toMatchInlineSnapshot` is an updateable snapshot (run `jest -u` to update it) so it should not be a variable reference.

Reply:
Sorry, should have dug in to that to do it correctly, but unfortunately I don't know  jest.  Thanks for fixing.

Reply:
I figured; no need to apologise, the comment was just for guidance.